NEWS.md

```diff
@@ -2,7 +2,7 @@
 
 ## 0.8.29 (in development)
 
-* Update company and product name for rebranding to Posit.
+* Update company and product names for rebranding to Posit.
```

Comment:
Let's add an additional NEWS entry for this PR, since we're rewriting hostnames. That's a functional change, and different from some of the earlier work.